### ISSUE TITLE
Idea: Eddie's Streaming Setup

### ISSUE THREAD

**Issue by mohyddintash** [EddieHubCommunity/support#535]
**Description**

A brief description of the question or issue: I find the setup really nice so It would be great if @eddiejaoude could make a video or an article about his streaming setup sharing details of the software and hardware used and how it's setup.

**Comment by eddiejaoude** [comment]
@mohyddintash you are right, this will be interesting for lots of people 👍 . I will do it in January 🎉 

thank you for raising the issue 🤓

**Comment by BOLT04** [comment]
Still interesting stale bot

**Comment by eddiejaoude** [comment]
I really want to do this, I am just thinking about best to record this 🤔 

- talking with b-roll of items?

**Comment by krishdevdb** [comment]
Someone, please reopen this?

**Comment by eddiejaoude** [comment]
I wrote a small blog on this https://www.eddiejaoude.io/blog-behind-the-scenes-desk-home-edition

I was going to do a video, but my setup has changed so much now I am full digital nomad for long periods of time - I can do an updated version which will be more focused on portable setup
